components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -279,12 +319,27 @@ public function front( $content ) {
 		// now use other methods in class to build array to search in/ use
 		$possible_pods = $this->auto_pods();
 
+		// build Pods object for current item
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 13983d28fa28e9f4c2d4da8567595959</sub>